### DISCUSSION TITLE
redirecting to a url in a new tab

### DISCUSSION THREAD

**Discussion by NatBLida68** [livewire/livewire#2644] [1. Help]
how can i redirect  wire click to a `new tab `  with a url

**Comment by bhaustein** [comment]
I think that would be an absolutely nice feature. As the response allways is parsed and interpreted by a javascript library of livewire it should be absolutely possible to be implemented. I would suggest something like this:

```php

function someClickHandler() {
    // ...do some work..
    $this->redirect('https://google.de', '_blank');
    // or
    return $this->redirect('https://google.de')->target('_blank');  // but this would be a breaking change...
}

```

**Reply by BishwajitAmali** [reply]
Why don't You use href directly , if you wants to redirect to new url ?

**Reply by bhaustein** [reply]
A usual task is that a click has some work for the server todo and after finishing that work you want to open a new url (and sometimes in a new tab)

An example from the real live is a "book now"-button. You book a ticket via API and get a temporary url of the new ticket back from the api. After finishing the booking you want to open the url of that ticket in a new tab while refreshing the active html document with some information about the successful payment.
